### PR TITLE
Adding docker hub malicious container findings

### PR DESCRIPTION
**Pull request by Caze121**
Signed-off-by: Ryan Gibbons <ryanjgibbons@hotmail.com>

### CODE REVIEW

**Review by szh — COMMENTED** [review]
This is great! Just put a couple tiny suggestions.

**Review by lumjjb — APPROVED** [review]
LGTM, after resolving @szh 's comments and passing CI

**Comment by szh** [comment]
@lumjjb I think a maintainer needs to approve running one of the GH workflows.

File: supply-chain-security/compromises/2022/docker-hub-malicious-containers.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!-- cspell:ignore Sysdig -->
+
+# Researchers at Sysdig find over 1650 malicious containers in public Docker Hub
+
+Docker Hub is the largest library and community for container images.
+
+## Impact
+
+There are hundreds of thousands of publicly available container images at the freely available Docker Hub library.  Researchers at Sysdig scanned over 250,000 unverified Linux images and identified 1,652 that were mailicious.  Crypto-miners represented the largest category, though significant numbers of embedded secrets and proxy avoidance tools were also found.
```
Comment (PushkarJ) [inline supply-chain-security/compromises/2022/docker-hub-malicious-containers.md:9]:
```suggestion
There are hundreds of thousands of publicly available container images at the freely available Docker Hub library.  Researchers at Sysdig scanned over 250,000 unverified Linux images and identified 1,652 that were malicious.  Crypto-miners represented the largest category, though significant numbers of embedded secrets and proxy avoidance tools were also found.
```

**Pull request merged**